### PR TITLE
Distraction fix

### PR DESCRIPTION
**Pull request by HussainGynai**
## Description
A fix for distraction

### Test Case 1
1. Set up game to be in attack goal situation
2. Run game with distraction
Expected result:  Passes shouldn't be too close and the pass locations shouldn't be the same every time.

### CODE REVIEW

**Review by chachmu — APPROVED** [review]
:+1: LGTM, not sure if this is the place to stylize clear.py but I don't think it really matters

File: soccer/gameplay/plays/Attack_Goal/distraction_attack.py [hunk 1]
```diff
@@ -230,11 +230,12 @@ def on_enter_passing(self):
     def on_enter_cross(self):
         self.remove_all_subbehaviors()
         #if the ball is passed to the distractor the ball is passed to the striker, as the third robot moves to the right to distract more
-        #add chip here if pass chance is low
-        self.add_subbehavior(
-            tactics.coordinated_pass.CoordinatedPass(self.striker_point),
-            'pass to striker',
-            required=True)
+        ball = main.ball().pos
+        self.striker_point.x = -(ball.x / 2 + constants.Field.Width / 4)
```
Comment (tjones320) [inline soccer/gameplay/plays/Attack_Goal/distraction_attack.py:234]: Do you always want this to happen on the right side of the field? You may want to reflect the play depending on robot positions.

**Review by tjones320 — CHANGES_REQUESTED** [review]

File: soccer/gameplay/plays/Attack_Goal/distraction_attack.py [hunk 2]
```diff
@@ -230,11 +230,12 @@ def on_enter_passing(self):
...
+        self.add_subbehavior(tactics.coordinated_pass.CoordinatedPass(
+            self.striker_point, use_chipper=True),
```
Comment (liquidmetal9015) [inline soccer/gameplay/plays/Attack_Goal/distraction_attack.py:236]: Are we confident that using the chipper for passes is going to work here? I guess it should if they are always far enough apart I just notice that the old pass was a grounded pass.

File: soccer/gameplay/plays/Attack_Goal/distraction_attack.py (hunk 1 above)
Comment (kylestach) [reply soccer/gameplay/plays/Attack_Goal/distraction_attack.py:234]: I would suggest some randomness here, but at the very least pick which side to go on in an intelligent fashion. Although, we should really integrate pass optimization here in the long run.

**Review by kylestach — APPROVED** [review]

**Review by liquidmetal9015 — APPROVED** [review]

**Pull request closed without merging**